test/set-schema-options.js

```diff
@@ -17,4 +17,17 @@ describe('set-schema-options', function() {
 
     setSchemaOptions(testSchema);
   });
+
+  describe("customOpts", function () {
+    it("should not set `ttlDuration` on options, for passed schema", function () {
+      var TTL_DURATION = 10000;
+      var testSchema = new Schema({ name: String, date: Date });
+      var options = { customOpts: { ttlDuration: TTL_DURATION }, option: true };
+
+      setSchemaOptions(testSchema, options);
+
+      expect(testSchema.get("ttlDuration")).to.equal(undefined);
```

Comment:
here, shouldn't you test whether customOpts is excluded?
```suggestion
      expect(testSchema.get("customOpts")).to.equal(undefined);
```

Reply:
my bad, missed this change when added the custom opts on top of ttl duration